runtime/tests/checker/path_test.go

```diff
@@ -105,3 +105,62 @@ func TestCheckPath(t *testing.T) {
 		assert.IsType(t, &sema.InvalidPathDomainError{}, errs[0])
 	})
 }
+
+func TestConvertStringToPath(t *testing.T) {
```

Comment:
Name differs from what is on master:

```suggestion
func TestCheckConvertStringToPath(t *testing.T) {
```